server/plugin.go

```diff
@@ -216,7 +217,8 @@ func (p *Plugin) handleClientID(w http.ResponseWriter, r *http.Request) {
 	userID := r.Header.Get("Mattermost-User-Id")
 
 	if userID == "" {
-		http.Error(w, "Not authorized", http.StatusUnauthorized)
+		p.API.LogWarn("Cannot fetch Client ID. Missing 'Mattermost-User-Id' header")
+		http.Error(w, "Cannot fetch Client ID. Missing 'Mattermost-User-Id' header.", http.StatusUnauthorized)
```

Comment:
The same changes as for `p.handleAuthorizeInADD` would also be valuable here. But this is not blocking.

Reply:
It's definitely a good idea. I made the same changes to more functions. What do you think about this?